service/src/main/java/uk/gov/mca/beacons/api/accountholder/mappers/AccountHolderMapper.java

```diff
@@ -38,6 +38,8 @@ public AccountHolder fromDTO(AccountHolderDTO accountHolderDTO) {
   public AccountHolder fromDTO(UpdateAccountHolderDTO dto) {
     final var attributes = dto.getAttributes();
     AccountHolder accountHolder = new AccountHolder();
+
+    accountHolder.setAuthId(dto.getAuthId());
```

Comment:
I feel that AuthId should never be updated. 

This is what ties the user in the db to the user in Azure, and this could cause a ton if issues if done incorrectly. 

Additionally, opens up a heap of questions about workflows

Reply:
Displayed in the backoffice ...maybe, but never updated.

Reply:
Similarly to how we don't display or allow updates to accountHolderId